saleor/graphql/webhook/subscription_types.py

```diff
@@ -1351,8 +1443,9 @@ def resolve_translation(root, _info: ResolveInfo):
         return translation
 
 
-class TranslationCreated(ObjectType, TranslationBase):
+class TranslationCreated(SubscriptionObjectType, TranslationBase):
     class Meta:
+        dry_run_model_name = "Translation"
```

Comment:
This one will not work, because root type depends of model wich was translated, ie. `AttributeTranslation` instead of `Translation`. Should we skip it and treat like `TYPE_NOT_SUPPORTED` or write some additional logic?

Reply:
For now I would skip it and consider adding support for it later.